HW7/vasia/Task7_3.py

```diff
@@ -0,0 +1,10 @@
+str = input("Enter a string")
+def num_characters():
+    '''This function calculates the number of characters included in a given string.'''
+    d={}
+    for i in range(len(str)):
+        d1={str[i]:str.count(str[i])}
+        d.update(d1)
+    print(d)
+    
+num_characters()
```

Comment:
here is another way:

def count_symbols_in_row(word):
    result = {}
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result